### DISCUSSION TITLE
How to run an application on the web using the https protocol

### DISCUSSION THREAD

**Discussion by RRozi** [flet-dev/flet#3221] [Q&A]
### Question

I heard that you need an SSL certificate. But I don't know anything about it

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Looks like a general question not related to Flet. Internet is full of guides how to do that. Most of the hosting providers already provide SSL for free.

**Reply by RRozi** [reply]
So the point is that my hosting doesn’t care what protocol the web server will be launched on, but the protocol is already setting up the web server. And I don’t even understand how to do it all
